packages/react-components/react-calendar-compat/library/src/components/Calendar/Calendar.tsx

```diff
@@ -340,19 +340,13 @@ export const Calendar: React.FunctionComponent<CalendarProps> = React.forwardRef
     const selectionAndTodayString = selectedDateString + ', ' + todayDateString;
 
     return (
-      <div
-        id={id}
-        ref={forwardedRef}
-        role="group"
-        aria-label={selectionAndTodayString}
-        className={classes.root}
-        onKeyDown={onDatePickerPopupKeyDown}
-      >
+      <div id={id} ref={forwardedRef} className={classes.root} onKeyDown={onDatePickerPopupKeyDown}>
         <div className={classes.liveRegion} aria-live="polite" aria-atomic="true">
           <span>{selectedDateString}</span>
         </div>
         {isDayPickerVisible && (
           <CalendarDay
+            ariaLabel={selectionAndTodayString}
```

Comment:
Seems like CI is complaining about this, maybe it is `aria-label`?

```suggestion
            aria-label={selectionAndTodayString}
```

Reply:
yup, we can only use kebab cased aria attributes